lib/transmission-rss/aggregator.rb

```diff
@@ -44,7 +44,13 @@ def run(interval = 600)
           @log.debug('aggregate ' + feed.url)
 
           begin
-            content = open(feed.url, allow_redirections: :safe).read
+            if feed.validate_cert == false
+              @log.debug('aggregate validate certificate: false')
+              content = open(feed.url, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE, allow_redirections: :safe}).read
```

Comment:
Could you assemble the options like in this example, please:

    options = {allow_redirections: :safe}

    unless feed.validate_cert
      options[:ssl_verify_mode] = OpenSSL::SSL::VERIFY_NONE
    end

    content = open(feed.url, options).read